m4_sessions/1711-inning/Group_3.md

```diff
@@ -1,9 +1,18 @@
 Group Three
 
-Session Title: Description: Why should students come to this session?
+Session Title: Ethics of Open Source
 
-Session Outcomes: What will students leave with or gain from this session?
+Description: The entire software industry is built on open source, and licensing can be confusing. We will look at some of the key ideas that have shaped open source, and the trajectory of open source. We will talk about how to ensure a sustainable open source ecosystem, and how to help developers continue to maintain that ecosystem.
+
+Session Outcomes: What will students leave with or gain from this session? A better understanding of many of the tools we all use everyday, and how important it is that those tools continue to be available.
 
 Session Outline: Include the flow of the session and the time for each section.
 
-Names of group members:
+15 mins Icebreaker/intros
+10 mins article summary of open source ideas and problems
+15 mins discussion
+15 mins how to contribute
+10 mins the future of open source
```

Comment:
Make sure to divide out who will be doing what part of this and have some specific talking points for each section.